terraform_files/baremetal_host/main.tf

```
@@ -68,8 +69,10 @@ resource "libvirt_domain" "host" {
     }
   }
 
+  machine = "q35"
+
   xml {
-    xslt = file("consolemodel.xsl")
+    xslt = file("console_cdrom_model.xsl")
```

```suggestion
    xslt = file("../baremetal_host/console_cdrom_model.xsl")
```
then those other files under ``terraform_files/baremetal``, ``terraform_files/none`` & ``terraform_files/baremetal_infra_env`` can safely be deleted and we can save ourselves the need to update it everywhere

---

done